grails-app/views/occurrence/_map.gsp

```diff
@@ -435,8 +435,10 @@ a.colour-by-legend-toggle {
         </p>
 
         <div class="recordLink">
-            <a href="#" class="erk-button erk-button--light">
-                <g:message code="search.map.popup.viewRecord" default="View record"/>
+            <a href="#">
```

Comment:
Well, it's a link that is shown as button. Existing script updates href after the popup is created. We could, of course, replace every one of those with click handler that changes location, but I'm not sure which solution is worse.